db/schema.rb

```diff
@@ -183,7 +183,7 @@
     t.string   "logo_name"
     t.string   "contact_email"
     t.integer  "org_type",               default: 0,              null: false
-    t.text     "links",                  default: "{\"org\":[]}"
+    t.text     "links"
```

Comment:
Should remove the default templates as well.

Reply:
good catch, thanks